src/components/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style: none;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+input:focus{
+    outline: none;
+}
+
+#topDiv{
+    border: 1px solid grey;
+    border-radius: 5px;
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 2%;
+
+}
+#buttomDiv{
+    border: 1px solid #fff;
+    border-radius: 5px;
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 4%;
+}
+
+#headInput{
+    font-weight: bold;
+    font-size: 18px;
+    width: 98%;
```

Comment:
width: 98% gibi input'u o alana sığdırmak için genişliğini daratmak için düşürdüğün % istediğin sonucu almak için uygun değil. Farklı ekran çözünürlüklerinde istenmeyen sonuçlara sebep olacaktır. Buna benzer kullandığın % lerin hepsini farklı ekran çözünürlülüklerinde kontrol etmelisin. %100 den %1 - %2 düşürmek yerine alternatif yöntemleri araştırmalısın.